src/Microsoft.AspNetCore.Cors/Infrastructure/CorsOptions.cs

```diff
@@ -32,6 +32,48 @@ public string DefaultPolicyName
         }
 
         /// <summary>
+        /// Adds a new policy and sets it as the default.
+        /// </summary>
+        /// <param name="name">The name of the policy.</param>
+        /// <param name="policy">The <see cref="CorsPolicy"/> policy to be added.</param>
+        public void AddDefaultPolicy(string name, CorsPolicy policy)
```

Comment:
Wouldn't we just use `DefaultPolicyName` here? Essentially have this method signature be `AddDefaultPolicy(CorsPolicy policy)` which internally calls `AddPolicy(DefaultPolicy, policy)`

Reply:
But this would require 2 lines of code (like setting the DefaultPolicyName and then doing AddDefaultPolicy(CorsPolicy)) and it implies an order too

Reply:
Don't we assign it a default name anyway? I'm not super familiar with these pieces, but I'd imagine a code path where I don't care about the name at all:

```
AddDefaultPolicy(new CorsPolicy { .. })

[EnableCore]
public IActionResult Foo()
```

^ No names involved.

Reply:
Yeah I don't think you need the user to pass in a name here... unless I'm also missing something.

Reply:
Though I admit it's odd to allow setting the default policy name. The default *should* have been some magical readonly token (or empty string or something).